include/logger/logger.hpp

```diff
@@ -58,29 +58,34 @@ class Logger {
         sLogLevel = level;
     }
 
+protected:
+    static constexpr auto cColorModule = "\033[34m";
+    static constexpr auto cColorNone   = "\033[0m";
+
+    static bool sColored;
+
 private:
     static constexpr auto cColorTime    = "\033[90m";
     static constexpr auto cColorDebug   = "\033[37m";
     static constexpr auto cColorInfo    = "\033[32m";
     static constexpr auto cColorWarning = "\033[33m";
     static constexpr auto cColorError   = "\033[31m";
     static constexpr auto cColorUnknown = "\033[36m";
-    static constexpr auto cColorModule  = "\033[34m";
```

Comment:
Move all the color consts to protected as well as make `GetLogLevel`, `GetCurrentTime` and `GetModule` protected and virtual. It will allow to override log formats in general.